src/timeseriesflattener/flattened_dataset.py

```diff
@@ -519,14 +538,14 @@ def _add_temporal_batch(  # pylint: disable=too-many-branches
 
         n_workers = min(self.n_workers, len(temporal_batch))
 
-        log.info(
-            f"Processing {len(temporal_batch)} temporal features in parallel with {n_workers} workers",
-        )
-
         # Chunksize is the number of predictors to process in each worker.
         # If we don't set chunksize, imap uses the default of 1, which means a bunch of IO overhead.
         chunksize = max(1, round(len(temporal_batch) / (n_workers)))
 
+        log.info(
```

Comment:
I'd remove everything after the first line from the info log and set it as a comment instead. We don't need to explain _that_ much to the user everytime